doc/Overview/Mvux/Overview.md

```diff
@@ -206,3 +214,33 @@ In order to summarize what we've just seen, let's return to the list of challeng
 - Feeds include various dimensions such as loading, if there's data or if an error occurred.
 - Feeds borrow from Option concept in functional programming where no data is a valid state for the feed.
 - MVUX combines the unidirectional flow of data, and immutability of MVU, with the data binding capabilities of MVVM.
+
+## Learning MVUX by Creating your own
+
+To create a new project using MVUX on Uno Platform see [How to set up an MVUX project](https://github.com/unoplatform/uno.extensions/blob/main/doc/Overview/Mvux/Tutorials/HowTo-MvuxProject.md).
+
+Then you can use the example above as a reference to create your own solution.
+
+As summarized below.
+
+### In the Model
+
+- Create you own Models
+- MVUX recommends using plain POCO (Plain Old CLR Object) record types for the models in your app as they're immutable.
```

Comment:
```suggestion
- MVUX as a pattern typically employs [Plain old CLR object](https://en.wikipedia.org/wiki/Plain_old_CLR_object) ('POCO') record types for the models in your app because of their immutability.
```

Reply:
```suggestion
- MVUX recommends using record types for the Models in your app as they're immutable.
```